### DISCUSSION TITLE
Will you continue to develop and maintain this library?

### DISCUSSION THREAD

**Discussion by pogadev18** [iway1/react-ts-form#52] [Q&A]
I recently discovered this repo after I started working on something similar for myself. I think what you've done so far is very well thought out and can be nicely developed in the medium to long term.

Yes, there are some limitations and I will come back with more feedback in the near future after I allocate more time to play around with the features you've built.

My question is, do you plan to continue developing this "form builder" and expand it with new features?

**Comment by iway1** [comment]
I intend to continue developing this library, specifically I would like to expand this libraries use case coverage from where it's at right now so that's it's usable in more situations. 

I don't necessarily think that it needs a large range of new features - this library is intended to make it very easy to build common types of form.

The big, obvious improvements that could be added right now IMO are:

1. Supporting more complex layouts easily (see https://github.com/iway1/react-ts-form/issues/54)
2. Dependent field values (see https://github.com/iway1/react-ts-form/issues/21)

This types of features would expand the libraries use case coverage significantly, while not bloating the API. So, these are the sorts of things I would be looking to add going forward. But, again, am open to ideas.

**Reply by pogadev18** [reply]
Yeah, makes sense!

Nice, well done and looking forward for this improvments!

Thanks for your reply
